editor/spatial_editor_gizmos.cpp

```diff
@@ -299,13 +299,24 @@ void EditorSpatialGizmo::add_handles(const Vector<Vector3> &p_handles, bool p_bi
 	}
 }
 
-void EditorSpatialGizmo::add_solid_box(Ref<Material> &p_material, Vector3 p_size) {
+void EditorSpatialGizmo::add_solid_box(Ref<Material> &p_material, Vector3 p_size, Vector3 p_position) {
 	ERR_FAIL_COND(!spatial_node);
 
 	CubeMesh cubem;
 	cubem.set_size(p_size);
+
+	Array arrays =  cubem.surface_get_arrays(0);
```

Comment:
There's a code style issue here because you used two spaces instead of one after the `=` operator.